examples/jobs/httpd.yaml

```diff
@@ -0,0 +1,15 @@
+name: httpd
+containers:
+- image: centos/httpd
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+jobs:
+- containers:
```

Comment:
I still don't see it :-( 

One is containers for the application (Deployment) the second one if for jobs. 
Jobs has also containers, its completely different set of containers from the top level containers.